docs/.vuepress/theme/components/Footer.vue

```diff
@@ -7,31 +7,11 @@
       <li><a href="https://github.com/ethereum" target="_blank">GitHub</a></li>
       <li><a href="https://twitter.com/ethereum" target="_blank">Twitter</a></li>
       <li><a href="https://blog.ethereum.org/">Blog</a></li>
-      <!-- <li><a href="#">Branding</a></li> -->
       <li><router-link to="/privacy-policy/">Privacy Policy</router-link></li>
       <li><router-link to="/terms-of-use/">Terms of Use</router-link></li>
       <li><router-link to="/cookie-policy/">Cookie Policy</router-link></li>
       <li><a href="mailto:press@ethereum.org">Contact</a></li>
     </ul>
-    <!-- Matomo -->
```

Comment:
This script was was causing errors in my Netlify previews. Added it back here (using the config): https://github.com/ethereum/ethereum-org-website/pull/159